react-props/my-app/src/ButtonContainer.tsx

```diff
@@ -0,0 +1,11 @@
+type Props = {
+  buttonContainerText: string;
+};
+
+export function ButtonContainer(props: Props) {
```

Comment:
This is a perfectly reasonable way to declare your props, but I suggest you get use to using the destructuring method since, once you get used to it, it's easier to read and work with.